eox-core-v-1-0/prose/edit/src/design-considerations-01-construction-principles.md

```diff
@@ -1,10 +1,32 @@
 ## Construction Principles
 
-TODO:
+An End-of-X (EOX) lifecycle statement represented by an OpenEoX Core document is created through the application of
```

Comment:
When we talk about OpenEoX Core, we need to avoid "document" and "statement":

OpenEoX statement: OpenEoX Core information bound to products
"documents": contains one or multiple statements

Reply:
Thank you! Suggestion applied and corrected!